src/manifest/manifest_test.go

```diff
@@ -102,9 +103,12 @@ func TestManifestValidateEmpty(t *testing.T) {
 	assert.NotNil(t, err)
 	expected := `manifest is not valid:
 	- key="version", value="0", failed "required" validation
-	- key="project", value="<nil>", failed "required" validation
+	- key="project.id", value="0", failed "required" validation
```

Comment:
Kedze uz nie je pointer, ale hodnota, ... ktora je vzdy zadana - aj ked prazdna -> tak validacia spadne o uroven nizsie.

Reply:
Realne by sa to nemalo nikdy stat, aby user videl tieto hlasky. Jedine ze rucne zmaze nieco z manifestu.